lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h

```diff
@@ -38,15 +40,15 @@ void vprintf_array(Value thread, ArrayRef<Value> arr, std::string info,
 // TODO(Superjomn): remove the code when MLIR v15.0 is included.
```

Comment:
wondering can we remove the `FuncOpConversionBase` code? I quickly checked MLIR main and it seems `FuncOpConversionBase` is not in header files. cc: @Superjomn

Reply:
> wondering can we remove the `FuncOpConversionBase` code? I quickly checked MLIR main and it seems `FuncOpConversionBase` is not in header files. cc: @Superjomn

Yeah, free to go if needed.